templates/_transkribus_options.html.twig

```diff
@@ -0,0 +1,23 @@
+<fieldset id="transkribus-line-ids" class="engine-options {% if engine != 'transkribus' %}hidden{% endif %}">
+    <legend>{{ msg('transkribus-options') }}</legend>
+    <div class="form-group">
+        <label for="lineId">{{ msg('transkribus-line-label') }}</label>
+        <select
+            name="lineId"
+            id="line-id"
+            class="form-control"
+            data-placeholder="Choose a line detection model"
+        >
+            <option value="0" {% if lineId == 0 %}selected{% endif %}>None</option>
+            <option value="49272" {% if lineId == 49272 %}selected{% endif %}>Default line detection model</option>
```

Comment:
Here and above in `app.js` it's adding these two options, but I'm confused about what the difference is between *not* sending a line ID and sending what we know to be the *default*. This looks like it'd be fine for now (although unnecessary), but if the default ever changes then this will be incorrect. Alternatively, 49272 is not applied unless we send it, but in that case we shouldn't be calling it 'default'.

Also, the text here and for the placeholder should be translated.

Reply:
It's more of a semantic error on my part 😅. No model being selected implies we let Transkribus choose the model automatically, whereas using the default model (49272) implies using the _public model that works best in most cases_, as suggested by Transkribus. 

[According to Transkribus](https://transkribus.eu/TrpServer/rest/models/layout/49272), model 49272 is named _Mixed Line Orientation_. Can we go ahead and use the same name?

Reply:
Ah that makes sense, and yep that name sounds fine (although I wonder how it'll translate… it's no worse than the PSM names though).

Reply:
Okayy